solana/solana-ibc/programs/solana-ibc/src/execution_context.rs

```diff
@@ -237,14 +244,16 @@ impl ibc::ExecutionContext for IbcStorage<'_, '_> {
         let port_channel = trie_ids::PortChannelPK::try_from(&path.0, &path.1)?;
         let trie_key = trie_ids::TrieKey::for_channel_end(&port_channel);
         let mut store = self.borrow_mut();
-        let digest = store
+        let encoded_channel_end = channel_end.clone().encode_vec();
+        let hash = CryptoHash::digest(encoded_channel_end.as_slice());
+        store
             .private
             .port_channel
             .entry(port_channel)
             .or_insert_with(Default::default)
             .set_channel_end(&channel_end)
```

Comment:
Should also update `PortChannelStore::set_channel_end` so that it doesn’t compute and return the hash since that’s no longer needed.

Reply:
gotcha